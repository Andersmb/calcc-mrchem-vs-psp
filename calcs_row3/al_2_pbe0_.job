#!/bin/bash

#SBATCH --account=nn4654k
#SBATCH --mail-type=None
#SBATCH --job-name=al_2_pbe0_
#SBATCH --output=al_2_pbe0_.log
#SBATCH --error=al_2_pbe0_.err
#SBATCH --time=00-48:00:00
#SBATCH --mem=160GB
#SBATCH --ntasks=30
#SBATCH --cpus-per-task=6

module purge
module load foss/2021b
module load OpenMPI/4.0.3-GCC-9.3.0
module load Python/3.8.2-GCCcore-9.3.0

cp al_2_pbe0_.inp $SCRATCH
cd $SCRATCH
export OMP_NUM_THREADS=6
$HOME/software/mrchem/install_upstream_master/bin/mrchem --launcher 'srun -n 30'  al_2_pbe0_

savefile al_2_pbe0_.out
savefile al_2_pbe0_.json

DIR=/cluster/projects/nn4654k/$(whoami)/MWOrbitals/${SLURM_JOBID}
mkdir -p $DIR
cp orbitals/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/al_2_pbe0_.orbitals

DIR=/cluster/projects/nn4654k/$(whoami)/MWCheckpoints/${SLURM_JOBID}
mkdir -p $DIR
cp checkpoint/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/al_2_pbe0_.checkpoint

exit 0