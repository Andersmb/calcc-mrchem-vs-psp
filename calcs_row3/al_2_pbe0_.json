{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.11213551993913855
          ],
          "radius": 1.82
        },
        {
          "center": [
            0.0,
            0.0,
            3.0017549700608614
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "al",
          "xyz": [
            0.0,
            0.0,
            -0.11213551993913855
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            3.0017549700608614
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "al_2_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        1.910253308142616e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.020660280611967882,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            5.0847153107021664e-08,
            -5.457672294094171e-08,
            -0.020660280611833226
          ],
          "vector_el": [
            5.076913456584634e-08,
            -5.465465989457769e-08,
            -1.5646534914076053
          ],
          "vector_nuc": [
            7.801854117532439e-11,
            7.793695363598193e-11,
            1.5439932107957721
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Al",
          "xyz": [
            0.0,
            0.0,
            -0.11213551993913855
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            3.0017549700608614
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -56.16603864293415,
          -4.191605032714478,
          -2.7224812178102575,
          -2.722481210964952,
          -2.7152982546092046,
          -0.37666600611272544,
          -0.21419191430824208,
          -56.166038642934005,
          -4.1916050327172085,
          -2.722481217815032,
          -2.7224812109691077,
          -2.7152982546101456,
          -0.3766660061033302,
          -0.21419191433222157
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -138.21752455893505
      },
      "scf_energy": {
        "E_ee": 117.56610712566452,
        "E_eext": 0.0,
        "E_el": -247.04264554691096,
        "E_en": -588.23958231899,
        "E_kin": 242.46005309055232,
        "E_next": 0.0,
        "E_nn": 4.174841742748635,
        "E_nuc": 4.174841742748635,
        "E_tot": -242.86780380416232,
        "E_x": -4.604845464850204,
        "E_xc": -14.224377979287658,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 117.2044702078491,
        "E_eext": 0.0,
        "E_el": -245.67799520322782,
        "E_en": -584.5738351433945,
        "E_kin": 240.48968380451367,
        "E_next": 0.0,
        "E_nn": 4.174841742748635,
        "E_nuc": 4.174841742748635,
        "E_tot": -241.50315346047918,
        "E_x": -4.601423295380308,
        "E_xc": -14.196890776815826,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 117.17478402855033,
              "E_eext": 0.0,
              "E_el": -246.99470617381024,
              "E_en": -585.3025609123039,
              "E_kin": 239.9060153091219,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.8198644310616,
              "E_x": -4.593418311336965,
              "E_xc": -14.179526287841677,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.8198644310616,
            "energy_update": 1.3167109705824203,
            "mo_residual": 0.1605465404763535,
            "wall_time": 79.450111481
          },
          {
            "energy_terms": {
              "E_ee": 117.32705546070434,
              "E_eext": 0.0,
              "E_el": -247.0350830859355,
              "E_en": -586.8316921282766,
              "E_kin": 241.2662988954223,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86024134318686,
              "E_x": -4.597934448295381,
              "E_xc": -14.198810865490202,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86024134318686,
            "energy_update": 0.04037691212525374,
            "mo_residual": 0.039732477380016507,
            "wall_time": 52.01990507
          },
          {
            "energy_terms": {
              "E_ee": 117.44252938603604,
              "E_eext": 0.0,
              "E_el": -247.04077132838003,
              "E_en": -587.60223120423,
              "E_kin": 241.9323039865703,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.8659295856314,
              "E_x": -4.601426605870696,
              "E_xc": -14.21194689088573,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.8659295856314,
            "energy_update": 0.005688242444534808,
            "mo_residual": 0.019859220109182323,
            "wall_time": 47.825574751
          },
          {
            "energy_terms": {
              "E_ee": 117.53662682814489,
              "E_eext": 0.0,
              "E_el": -247.04222812493913,
              "E_en": -588.1284527293868,
              "E_kin": 242.37554366390737,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.8673863821905,
              "E_x": -4.604120482611114,
              "E_xc": -14.221825404993467,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.8673863821905,
            "energy_update": 0.0014567965590970289,
            "mo_residual": 0.016675722320701157,
            "wall_time": 49.27851539
          },
          {
            "energy_terms": {
              "E_ee": 117.5845346416453,
              "E_eext": 0.0,
              "E_el": -247.04256072994076,
              "E_en": -588.3818045921006,
              "E_kin": 242.586898007144,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86771898719212,
              "E_x": -4.6054683133830165,
              "E_xc": -14.226720473246422,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86771898719212,
            "energy_update": 0.0003326050016312365,
            "mo_residual": 0.011882788319083213,
            "wall_time": 48.750017533
          },
          {
            "energy_terms": {
              "E_ee": 117.57350882290636,
              "E_eext": 0.0,
              "E_el": -247.042616290892,
              "E_en": -588.2795463686942,
              "E_kin": 242.49347137479404,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86777454814336,
              "E_x": -4.604983964797396,
              "E_xc": -14.225066155100755,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86777454814336,
            "energy_update": 5.556095123893101e-05,
            "mo_residual": 0.006773452450679904,
            "wall_time": 47.02906159
          },
          {
            "energy_terms": {
              "E_ee": 117.56748203001298,
              "E_eext": 0.0,
              "E_el": -247.0426347732962,
              "E_en": -588.2157466116078,
              "E_kin": 242.43465798510593,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86779303054755,
              "E_x": -4.604809977647529,
              "E_xc": -14.224218199159765,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86779303054755,
            "energy_update": 1.8482404186670465e-05,
            "mo_residual": 0.003936106621563488,
            "wall_time": 46.608974321
          },
          {
            "energy_terms": {
              "E_ee": 117.56348929996736,
              "E_eext": 0.0,
              "E_el": -247.04264264035686,
              "E_en": -588.2142307533692,
              "E_kin": 242.43683589767645,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86780089760822,
              "E_x": -4.604747279111037,
              "E_xc": -14.2239898055205,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86780089760822,
            "energy_update": 7.867060674016102e-06,
            "mo_residual": 0.002952691274475715,
            "wall_time": 49.566816267
          },
          {
            "energy_terms": {
              "E_ee": 117.56353616728713,
              "E_eext": 0.0,
              "E_el": -247.04264453663157,
              "E_en": -588.2260964536072,
              "E_kin": 242.44880200510931,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86780279388293,
              "E_x": -4.604775288801178,
              "E_xc": -14.224110966619666,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86780279388293,
            "energy_update": 1.8962747105888411e-06,
            "mo_residual": 0.0010128316984679062,
            "wall_time": 51.40935625
          },
          {
            "energy_terms": {
              "E_ee": 117.56572930002378,
              "E_eext": 0.0,
              "E_el": -247.04264537754636,
              "E_en": -588.2403881051436,
              "E_kin": 242.46124967508737,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86780363479772,
              "E_x": -4.60485416276697,
              "E_xc": -14.22438208474703,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86780363479772,
            "energy_update": 8.409147937982198e-07,
            "mo_residual": 0.0007080865257725709,
            "wall_time": 52.360853032
          },
          {
            "energy_terms": {
              "E_ee": 117.56615316288149,
              "E_eext": 0.0,
              "E_el": -247.04264551915858,
              "E_en": -588.2415271286783,
              "E_kin": 242.4619868329011,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86780377640994,
              "E_x": -4.604854596490381,
              "E_xc": -14.224403789772474,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86780377640994,
            "energy_update": 1.416122188402369e-07,
            "mo_residual": 0.00020767435110871082,
            "wall_time": 48.368140321
          },
          {
            "energy_terms": {
              "E_ee": 117.56610712566452,
              "E_eext": 0.0,
              "E_el": -247.04264554691096,
              "E_en": -588.23958231899,
              "E_kin": 242.46005309055232,
              "E_next": 0.0,
              "E_nn": 4.174841742748635,
              "E_nuc": 4.174841742748635,
              "E_tot": -242.86780380416232,
              "E_x": -4.604845464850204,
              "E_xc": -14.224377979287658,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -242.86780380416232,
            "energy_update": 2.7752378173317993e-08,
            "mo_residual": 5.806904650467794e-05,
            "wall_time": 56.404291959
          }
        ],
        "wall_time": 629.13198309
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
