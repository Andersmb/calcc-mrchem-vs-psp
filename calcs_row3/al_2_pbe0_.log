Starting job 5688102 on c1-[30,53],c5-[2,7,36,60] at Thu May 19 19:24:39 CEST 2022

Copying al_2_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying al_2_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688102      al_2_pbe0_        180                                             00:13:47      0:0 
5688102.bat+      batch         36        1   00:00:02          0   00:00:02   00:13:47      0:0 
5688102.ext+     extern        180        6   00:00:00          3   00:00:00   00:13:47      0:0 
5688102.0      mrchem.x        180       30   00:13:36         29   00:26:34   00:13:40      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688102                                                                          
5688102.bat+     19328K          0     19328K        0              0          0 
5688102.ext+          0          3          0        0              3          0 
5688102.0      3807652K         29 2458055475        0             18          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688102                                                                                               
5688102.bat+       16.88M               0         16.88M        0.03M                0          0.03M 
5688102.ext+        0.00M               3          0.00M            0                3              0 
5688102.0           6.77M               0          6.74M       59.21M                0         26.90M 

Job 5688102 completed at Thu May 19 19:38:26 CEST 2022
