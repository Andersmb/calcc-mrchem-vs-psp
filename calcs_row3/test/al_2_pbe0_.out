                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -0.112136
    1       H            :        0.000000        0.000000        3.001755
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               7
 Beta electrons          :                                               7
 Total electrons         :                                              14
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000001530e+00
    1    1    a          :                              9.999999894007e-01
    2    1    a          :                              1.000000047021e+00
    3    1    a          :                              9.999999999742e-01
    4    1    a          :                              1.000000022181e+00
    5    1    a          :                              1.000000028240e+00
    6    1    a          :                              1.000000462691e+00
    7    1    b          :                              1.000000001530e+00
    8    1    b          :                              9.999999894007e-01
    9    1    b          :                              1.000000047021e+00
   10    1    b          :                              9.999999999742e-01
   11    1    b          :                              1.000000022181e+00
   12    1    b          :                              1.000000028240e+00
   13    1    b          :                              1.000000462691e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                240.489683804514
 E-N energy              :            (au)               -584.573835143394
 Coulomb energy          :            (au)                117.204470207849
 Exchange energy         :            (au)                 -4.601423295380
 X-C energy              :            (au)                -14.196890776816
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  4.174841742749
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -245.677995203228
 Nuclear energy          :            (au)                  4.174841742749
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.415031534605e+02
                         :      (kcal/mol)             -1.515455155898e+05
                         :        (kJ/mol)             -6.340664376393e+05
                         :            (eV)             -6.571635299602e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.790408171617
    1    1    a          :            (au)                 -4.168295710008
    2    1    a          :            (au)                 -2.695586122054
    3    1    a          :            (au)                 -2.695558670757
    4    1    a          :            (au)                 -2.688024835281
    5    1    a          :            (au)                 -0.365429814431
    6    1    a          :            (au)                 -0.198861857130
    7    1    b          :            (au)                -55.790408171617
    8    1    b          :            (au)                 -4.168295710008
    9    1    b          :            (au)                 -2.695586122054
   10    1    b          :            (au)                 -2.695558670757
   11    1    b          :            (au)                 -2.688024835281
   12    1    b          :            (au)                 -0.365429814431
   13    1    b          :            (au)                 -0.198861857130
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -137.204330362555
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          3.741657e+00        -241.503153460479         -2.415032e+02
    1          1.605465e-01        -242.819864431062         -1.316711e+00
    2          3.973248e-02        -242.860241343187         -4.037691e-02
    3          1.985922e-02        -242.865929585631         -5.688242e-03
    4          1.667572e-02        -242.867386382190         -1.456797e-03
    5          1.188279e-02        -242.867718987192         -3.326050e-04
    6          6.773452e-03        -242.867774548143         -5.556095e-05
    7          3.936107e-03        -242.867793030548         -1.848240e-05
    8          2.952691e-03        -242.867800897608         -7.867061e-06
    9          1.012832e-03        -242.867802793883         -1.896275e-06
   10          7.080865e-04        -242.867803634798         -8.409148e-07
   11          2.076744e-04        -242.867803776410         -1.416122e-07
   12          5.806905e-05        -242.867803804162         -2.775238e-08
---------------------------------------------------------------------------
                      SCF converged in 12 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -0.112136
    1       H            :        0.000000        0.000000        3.001755
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                242.460053090552
 E-N energy              :            (au)               -588.239582318990
 Coulomb energy          :            (au)                117.566107125665
 Exchange energy         :            (au)                 -4.604845464850
 X-C energy              :            (au)                -14.224377979288
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  4.174841742749
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -247.042645546911
 Nuclear energy          :            (au)                  4.174841742749
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.428678038042e+02
                         :      (kcal/mol)             -1.524018466023e+05
                         :        (kJ/mol)             -6.376493265981e+05
                         :            (eV)             -6.608769325563e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.166038642934
    1    1    a          :            (au)                 -4.191605032714
    2    1    a          :            (au)                 -2.722481217810
    3    1    a          :            (au)                 -2.722481210965
    4    1    a          :            (au)                 -2.715298254609
    5    1    a          :            (au)                 -0.376666006113
    6    1    a          :            (au)                 -0.214191914308
    7    1    b          :            (au)                -56.166038642934
    8    1    b          :            (au)                 -4.191605032717
    9    1    b          :            (au)                 -2.722481217815
   10    1    b          :            (au)                 -2.722481210969
   11    1    b          :            (au)                 -2.715298254610
   12    1    b          :            (au)                 -0.376666006103
   13    1    b          :            (au)                 -0.214191914332
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -138.217524558935
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000       -0.000000       -1.564653
 Magnitude               :            (au)                        1.564653
                         :         (Debye)                        3.976952
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.543993
 Magnitude               :            (au)                        1.543993
                         :         (Debye)                        3.924439
---------------------------------------------------------------------------
 Total vector            :        0.000000       -0.000000       -0.020660
 Magnitude               :            (au)                        0.020660
                         :         (Debye)                        0.052513
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 13m 36s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
